### PR TITLE
Vkarinje/24

### LINKED ISSUE #24: Player Creation
As a Player, I will create a player object, so that I can play the game.

### PR DESCRIPTION
**Pull request by vkarinje**
Added code for creating a player.

### CODE REVIEW

File: blankwindow.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import pygame, sys
 pygame.init()
-screen_width,screen_height=1280,720
-blank_screen=pygame.display.set_mode((screen_width,screen_height))
+window_width,window_height=1280,720
+display_surface=pygame.display.set_mode((screen_width,screen_height))
```
Comment (AkhilaChiluveru) [inline blankwindow.py]: Updated names make more sense.

File: blankwindow.py [hunk 2]
```diff
@@ -2,15 +2,33 @@
 pygame.init()
 window_width,window_height=1280,720
 display_surface=pygame.display.set_mode((window_width,window_height))
+
 #setting the screen title
 pygame.display.set_caption('Hit-Get')
 
+#importing images
+ship_surf = pygame.image.load('C:/Fall 2022/SE/hit-get/graphics/ship.png').convert_alpha()
```
Comment (AkhilaChiluveru) [inline blankwindow.py:11]: Good to see some kind of player. But any reason to choose a ship in specific?

**Review by AkhilaChiluveru — APPROVED** [review]

File: blankwindow.py (hunk 2 above)
Comment (vkarinje) [reply blankwindow.py:11]: As this image has pointed edge I felt it looks cool. Also, as I have designed the  game window background as space, I felt space ship would look cool.

**Review by AkhilaChiluveru — APPROVED** [review]
Fine. I see that you have selected a bullet and player. Looks good to me. Approving changes.

**Comment by AkhilaChiluveru** [comment]
Looks good. Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Looks good approved.

**Comment by AkhilaChiluveru** [comment]
n

**Review by AkhilaChiluveru — APPROVED** [review]

**Comment by AkhilaChiluveru** [comment]
File looks good to me

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Comment by AkhilaChiluveru** [comment]
Approved

**Review by AkhilaChiluveru — APPROVED** [review]

**Comment by AkhilaChiluveru** [comment]
Approved

**Review by AkhilaChiluveru — APPROVED** [review]

**Comment by AkhilaChiluveru** [comment]
Approved

**Review by AkhilaChiluveru — APPROVED** [review]

**Comment by AkhilaChiluveru** [comment]
Approved

**Review by AkhilaChiluveru — APPROVED** [review]

**Review by AkhilaChiluveru — APPROVED** [review]

**Comment by AkhilaChiluveru** [comment]
Approved

**Review by AkhilaChiluveru — APPROVED** [review]
Everything looks good

**Review by AkhilaChiluveru — APPROVED** [review]

**Comment by AkhilaChiluveru** [comment]
Template shooter, targets and laser added. Apprving changes

**Pull request merged**